core/jobmanager.py

```diff
@@ -42,7 +43,7 @@ class JobManager:
 
     """
 
-    def __init__(self, settings, path=None, folder=None, use_existing_folder=False):
+    def __init__(self, settings, path=None, folder=None, use_existing_folder=False, csv_logger: LoggerCSV = None):
```

Comment:
This gives you flexibility, because ideally if you want to log all the jobs in the `default_jobmanager` you can in principle provide that one as argument and therefore centralize the jobs logs in one file.